subxt/src/error.rs

```diff
@@ -190,5 +204,5 @@ pub struct ModuleError {
 pub trait HasModuleError {
     /// If the error has a `Module` variant, return a tuple of the
     /// pallet index and error index. Else, return `None`.
-    fn module_error_indices(&self) -> Option<(u8, u8)>;
+    fn module_error_indices(&self) -> Option<ModuleErrorRaw>;
```

Comment:
This method should be renamed depending on what name you choose for `ModuleErrorRaw`